src/Cli/dotnet/Commands/Test/ExitCode.cs

```diff
@@ -7,4 +7,5 @@ internal static class ExitCode
 {
     public const int Success = 0;
     public const int GenericFailure = 1;
+    public const int ZeroTests = 8;
```

Comment:
Why did we skip all the way to 8?

Reply:
These must match MTP:

https://github.com/microsoft/testfx/blob/main/src/Platform/Microsoft.Testing.Platform/Helpers/ExitCodes.cs

Reply:
I added a comment to the class